Userland/Applications/Calendar/main.cpp

```diff
@@ -53,7 +53,7 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         unsigned view_year = calendar->view_year();
         if (calendar->mode() == GUI::Calendar::Month) {
             view_month--;
-            if (calendar->view_month() == 1) {
+            if (calendar->view_month() == 0) {
```

Comment:
Would be nice if you could briefly explain in the commit description what sort of issue this caused :P